idv/frontend-vite-vue/src/components/HelloWorld.vue

```diff
@@ -14,6 +14,13 @@ export default {
         publicKey,
         l10n: { locale: "en-US" },
         onComplete: (validationToken) => {
+          // TODO: User has finished the flow. This validation token can be used to see the fp_id
```

Comment:
nit; instead of see, I would say fetch and you can even link to the relevant parts of the docs